infinitude

```diff
@@ -236,7 +236,9 @@ any '/api/*path' => sub {
 			$setting->{data} = { %{$search->TO_JSON()} };
 			$store->set('systems.xml',  $xml.'');
 			$store->set('systems.json', $xml->_as_json());
-			$store->set(changes => 'true');
+			if (('POST' eq $c->req->method) or ('PUT' eq $c->req->method)) { # Only set changes if it's a POST
+				$store->set(changes => 'true');
```

Comment:
This will likely break clients which are using GET to modify state (which is of course wrong wrong wrong, but I think some people are limited by their HA software). This is the correct behavior though, so thanks for the patch.